.github/issue_template/bug.md

```diff
@@ -0,0 +1,55 @@
+---
+name: 🐛 Bug Report
+about: Submit a bug report to help us improve
+labels: "bug"
+---
+
+<!--
+Note: if you are reporting a wrong signature of a function or a class in
+the standard library, then the typeshed tracker is better suited
+for this report: https://github.com/python/typeshed/issues
+-->
+
+## 🐛 Bug Report
+
+(A clear and concise description of what the bug is.)
+
+## To Reproduce
+
+(Write your steps here:)
+
+1. Step 1...
+1. Step 2...
+1. Step 3...
+
+## Expected Behavior
+
+<!--
+  How did you expect your project to behave?
+  It’s fine if you’re not sure your understanding is correct.
+  Write down what you thought would happen.
+-->
+
+(Write what you thought would happen.)
+
+## Actual Behavior
+
+<!--
+  Did something go wrong?
+  Is something broken, or not behaving as you expected?
+  Describe this section in detail, and attach screenshots if possible.
```

Comment:
I don't think for mypy screenshots are ever relevant. We don't want people to attach screenshots of error messages (that's quite counterproductive).